src/masternodes/mn_checks.cpp

```diff
@@ -820,6 +820,8 @@ class CCustomTxApplyVisitor : public CCustomTxVisitor {
         node.creationHeight      = height;
         node.operatorType        = obj.operatorType;
         node.operatorAuthAddress = obj.operatorAuthAddress;
+        node.rewardAddressType   = obj.rewardType;
```

Comment:
Needs fork guard.

Reply:
i've set the height to the NextNetworkUpdate since we don't have it set yet.